runtime/interpreter/errors.go

```diff
@@ -510,19 +510,17 @@ func (e InvocationArgumentTypeError) Error() string {
 	)
 }
 
-// InvocationReceiverTypeError
+// MemberAccessTypeError
 //
-type InvocationReceiverTypeError struct {
```

Comment:
This error was a leftover from a temporary period where we defensively checked an invocation's receiver (`v` in `v.f()`).

This PR is a generalization to any member access, reads and writes (`v.p` and `v.p = ...`), which includes also reads which are then invoked (`v.f()`).